bin/MadGraph5_aMCatNLO/cards/production/13p6TeV/WprimeToENu_LO/WprimeToENu_M-1000_kR-0p01_LO_NNPDF31nnlo/WprimeToENu_M-1000_kR-0p01_LO_NNPDF31nnlo_customizecards.dat

```diff
@@ -0,0 +1,3 @@
+set param_card mass 34 1000
+set param_card DECAY 34 0.003354589352653081
```

Comment:
how is this width chosen

Reply:
Hi @menglu21 , 
The width is calculated using the WprimeWidth.py script, which is included in the "WprimeToENu_LO" directory. This script outputs the decay width depending on the mass and coupling. For this particular case, mass of 1000 and coupling of 0.01 are used, which results in the width of 0.003354589352653081.